crates/prompt_gateway/src/prompt_stream_context.rs

```diff
@@ -1203,40 +1096,6 @@ impl HttpContext for StreamContext {
             };
         self.is_chat_completions_request = true;
```

Comment:
This is redundant. The stream has already been marked in headers.

Reply:
correct - removing it